<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Protocol_View</class>
 <widget class="QWidget" name="Protocol_View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="meas_splitter">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>10</number>
     </property>
     <property name="midLineWidth">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="sequence_main_widget" native="true">
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_remove_step">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
                                background-color:  #FF3333; 
                                color: white; 
                                border: none; 
                                padding: 0px; 
                                padding-bottom: 5px;
                                text-align: center; 
                                text-decoration: none; 
                                font-size: 18px; 
                                margin: 2px 2px; 
                                border-radius: 6px;
								font-weight: bold;
                            }

            QPushButton:hover {
                background-color: #B22222;
            }
QPushButton:disabled {
        background-color: #808080;
    }</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="toolButton_add_step">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
                                background-color: #4CAF50; 
                                color: white; 
                                border: none; 
                                padding: 0px; 
                                padding-bottom: 5px;
                                text-align: center; 
                                text-decoration: none; 
                                font-size: 18px; 
                                margin: 2px 2px; 
                                border-radius: 6px;
								font-weight: bold;
                            }

                            QToolButton:hover {
                                background-color: #45a049;
                            }</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_sequence">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>19</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Sequence</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_move_step_up">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2a4cdf; /* This is a lighter shade of blue */
    color: white;
    border: none;
    padding: 2px 10px;
    text-align: center;
    text-decoration: none;
    font-size: 13px;
    margin: 2px 2px;
    border-radius: 6px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1a3cbf; /* This is a darker shade of the lighter blue for hover effect */
}
QPushButton:disabled {
        background-color: #808080;
    }</string>
         </property>
         <property name="text">
          <string>move up ↑</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QTreeView" name="treeView_protocol_sequence"/>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_move_step_down">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2a4cdf; /* This is a lighter shade of blue */
    color: white;
    border: none;
    padding: 2px 10px;
    text-align: center;
    text-decoration: none;
    font-size: 13px;
    margin: 2px 2px;
    border-radius: 6px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1a3cbf; /* This is a darker shade of the lighter blue for hover effect */
}
QPushButton:disabled {
        background-color: #808080;
    }</string>
         </property>
         <property name="text">
          <string>move down ↓</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_move_step_out">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2a4cdf; /* This is a lighter shade of blue */
    color: white;
    border: none;
    padding: 2px 10px;
    text-align: center;
    text-decoration: none;
    font-size: 13px;
    margin: 2px 2px;
    border-radius: 6px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1a3cbf; /* This is a darker shade of the lighter blue for hover effect */
}
QPushButton:disabled {
        background-color: #808080;
    }</string>
         </property>
         <property name="text">
          <string>move out ←</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_move_step_in">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2a4cdf; /* This is a lighter shade of blue */
    color: white;
    border: none;
    padding: 2px 10px;
    text-align: center;
    text-decoration: none;
    font-size: 13px;
    margin: 2px 2px;
    border-radius: 6px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1a3cbf; /* This is a darker shade of the lighter blue for hover effect */
}
QPushButton:disabled {
        background-color: #808080;
    }</string>
         </property>
         <property name="text">
          <string>move in →</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="configuration_main_widget" native="true">
      <layout class="QGridLayout" name="gridLayout_9">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_configuration">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>19</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Configuration</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="loopstep_configuration_widget" native="true">
         <layout class="QGridLayout" name="gridLayout_10">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
